src/app/workflow/dag/wdl-viewer/state/wdl-viewer.model.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Defines types for the response of Pipeline Builder library
+ */
+export interface WdlViewerPipeline {
```

Comment:
I know it was like this already, but I would call this something like WdlViewerPipelineResponse or WdlViewerPipelineModel to clearly indicate this is not a pipeline.